frontend/src/metabase/common/components/Table/Table.unit.spec.tsx

```diff
@@ -304,6 +309,36 @@ describe("common > components > Table", () => {
       screen.getByRole("navigation", { name: /pagination/ }),
     ).toBeInTheDocument();
   });
+
+  it("should accept a format value function when sorting", async () => {
+    render(
+      <ClientSortableTable
+        columns={sampleColumns}
+        rows={sampleData}
+        rowRenderer={renderRow}
+        formatValueForSorting={(row, colName) => {
+          if (colName === "type") {
+            if (row.type === "Water") {
+              return 10;
+            }
+            return 1;
+          }
+          return row[colName as keyof Pokemon];
+        }}
+        locale="en-US"
+      />,
+    );
+    expect(screen.getByText("Name")).toBeInTheDocument();
+    expect(screen.getByText("Type")).toBeInTheDocument();
+    expect(screen.getByText("Generation")).toBeInTheDocument();
+
+    const sortNameButton = screen.getByText("Type");
+    // Ascending
+    await userEvent.click(sortNameButton);
+    // Descending
+    await userEvent.click(sortNameButton);
+    firstRowShouldHaveText("Squirtle");
+  });
```

Comment:
I think this is a copy of the test above it, though it's possible I've missed something